gradle/libs.versions.toml

```diff
@@ -254,10 +254,10 @@ androidx-core-ktx = "androidx.core:core-ktx:1.10.1"
 androidx-mediarouter = "androidx.mediarouter:mediarouter:1.7.0"
 androidx-palette = "androidx.palette:palette:1.0.0"
 androidx-preference-ktx = "androidx.preference:preference-ktx:1.2.1"
-androidx-recyclerview = "androidx.recyclerview:recyclerview:1.3.0"
+androidx-recyclerview = "androidx.recyclerview:recyclerview:1.3.2"
 androidx-swiperefreshlayout = "androidx.swiperefreshlayout:swiperefreshlayout:1.1.0"
 androidx-uiautomator = "androidx.test.uiautomator:uiautomator:2.2.0"
-androidx-viewpager = "androidx.viewpager2:viewpager2:1.1.0"
+androidx-viewpager = "androidx.viewpager2:viewpager2:1.1.0-beta02"
```

Comment:
Update `ViewPager2 `to `1.1.0-beta02 ` based on recycler view release notes guidelines:
https://developer.android.com/jetpack/androidx/releases/recyclerview#recyclerview-1.3.1-rc01

> Users of ViewPager2 must update to at least 1.1.0-beta02 when updating to this version of RecyclerView to avoid crashes.